summary_workflows/quantification/Q2_benchmark/Q2_benchmark_specification.md

```diff
@@ -39,59 +39,43 @@ To compute the metrics, the poly(A) sites identified from the RNAseq data using
+Fields:
 
 - **chrom** - the name of the chromosome
-- **chromStart** - the position of cleavage/polyadenylation in genomic coordinates
-- **chromEnd** - the position of cleavage/polyadenyalation in genomic coordinates
+- **chromStart** - the starting position of the feature in the chromosome; this corresponds to the nucleotide just upstream of the cleavage and polyadenylation reaction; the starting position is 0-based, i.e. the first base on the chromosome is numbered 0
+- **chromEnd** - the ending position of the feature in the chromosome; as identified PAS are single-nucleotide, the ending position is equal to `chromStart + 1`
 - **name** - defines the name of the identified poly(A) site
 - **score** - the TPM value - quantified PAS
-- **strand** - defines the strand; either "." (=no strand) or "+" or "-".
+- **strand** - defines the strand; either "." (=no strand) or "+" or "-"
 
 #### Format 2
 
 This BED file contains genomic positions of unique unique cleavage/polyadenylation sites as well as TPM values for each identified site quantified from the orthogonal 3'end-seq dataset.
-Fields (the same as format 1):
-
-- **chrom** - the name of the chromosome
-- **chromStart** - the position of cleavage/polyadenylation in genomic coordinates
-- **chromEnd** - the position of cleavage/polyadenylation in genomic coordinates
-- **name** - defines the name of the identified poly(A) site
-- **score** - the TPM value - quantified PAS
-- **strand** - defines the strand; either "." (=no strand) or "+" or "-".
-
-#### Note on Genomic Coordinates of Cleavage Sites
-
-The positions specified in the **chromStart** and **chromEnd** fields have the interpretation:
-
- - `0` == 5'-end of first nucleotide
- - `1` == 5'-end of second nucleotide == 3'-end of first nucleotide 
- - `n` == 5'-end of (n+1)-th nucleotide == 3'-end of n-th nucleotide
-
-Since cleavage sites occur between nucleotides, the specification of a cleavage site corresponds to the case where **chromStart** and **chromEnd** are identical.
+Fields are the same as in format 1.
 
 ## Outputs
 
 | # | Format | Link | Example data |
- | --- | --- | --- | --- |
- | 1 | JSON | [Specification][spec-json] | [Link][out1] |
- 
+| --- | --- | --- | --- |
+| 1 | JSON | [Specification][spec-json] | [Link][out1] |
+
 ### Additional info outputs
  
 #### Format 1
  
  The following table lists the attribute names, value types and units, and a
 description of each attribute-value pair:
  
- | Attribute | Type | Unit | Description |
- | --- | --- | --- | --- |
- | `correlation_coefficient` | `float` | N/A | Correlation between RNAseq-based quantification and 3'end-seq quantification |
+| Attribute | Type | Unit | Description |
+| --- | --- | --- | --- |
+| `correlation_coefficient` | `float` | N/A | Correlation between RNAseq-based quantification and 3'end-seq quantification |
 
```

Comment:
This file is also changed in another PR (https://github.com/iRNA-COSI/APAeval/pull/210/files#diff-752c23db2dc0d0c543326d5e5ca8bb7d288d17f8b2c70a7388b8332c585d6d7f) which will most certainly lead to conflicts, as the changes there are different from those here. Which are the "true" specifications?

Reply:
The file in this repo was last updated in November 2021 and, as you say, @daneckaw has been working on the specifications since then so we should give priority to the content in her PR #210.